sourabh/Programming_assignment4.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html>
+	<head>
+		<title>RUSUME</title>
+		<style>
+			table , th , td {
+			border: 2px solid black;}
+		</style>
+	</head> 
+	<body>
+		<h1><em>SOURABH  BHOJANE.</em></h1>
+			<small>bhojane.sourabh14@gmail.com</small>
+			<hr>
+		<h2>ABOUT ME :</h2>
+		<ul>
+			<li>My name is Sourabh Bhojane.</li>
+			<li>I am 22 years old.</li>
+			<li>I was born and bought up in pune and living in katraj.</li>
+			<li>I have completed my 10th from <strong> N.E.S.T SCHOOL.</strong> & 12th from <b> S.P Collage.</b></li>
+			<li>Appearing T.Y.Bcs in Abhinav education society.</li>
+			<li>My hobbies are playing footboll,swaming,& bike trips.</li>
+		</ul>
+		<h2>WORK :</h2>
+		<ul>
+			<li>I'm Fresher</li>
+		</ul>
+		<h2>Interests :</h2>
+		<ul>
+			<li>Playing cricket.</li>
+			<a href="https://en.wikipedia.org/wiki/Cricket" target="_blank"><img src="images.jpg"></a>
```

Comment:
this is wrong. there should be only one tag sibling to "li" tag, and that is another "li" tag.
Do you know about the child-parent-sibling relation in HTML?

Reply:
you must give your answer in the reply section, rather than resolving the comment.